pytorch_alternatives/custom_pytorch_nlp/Headline Classifier Local.ipynb

```diff
@@ -514,9 +478,9 @@
  "metadata": {
   "instance_type": "ml.t3.medium",
   "kernelspec": {
-   "display_name": "Python 3 (PyTorch 1.6 Python 3.6 CPU Optimized)",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "python3__SAGEMAKER_INTERNAL__arn:aws:sagemaker:ap-southeast-1:492261229750:image/pytorch-1.6-cpu-py36-ubuntu16.04-v1"
+   "name": "python3"
```

Comment:
I assume this is incidental to whatever environment you were working on, rather than actively intending to change the kernel specs for the notebooks - should be restored to previous